spec/classes/graphite_powershell_spec.rb

```diff
@@ -4,41 +4,48 @@
   context 'supported operating systems' do
     ['windows'].each do |osfamily|
       describe "graphite_powershell class without any parameters on #{osfamily}" do
-        let(:params) {{
-          :server => 'localhost'
-        }}
-        let(:facts) {{
-          :osfamily     => osfamily,
-          :architecture => 'amd64'
-        }}
+        let(:params) do
+          {
+            server: 'localhost'
+          }
+        end
+        let(:facts) do
+          {
+            osfamily: osfamily,
+            architecture: 'amd64'
+          }
+        end
 
         it { should contain_class('graphite_powershell::params') }
-        it { should contain_class('graphite_powershell::config').that_comes_before('graphite_powershell::install') }
-        it { should contain_class('graphite_powershell::install').that_comes_before('graphite_powershell::service') }
+        it { should contain_class('graphite_powershell::config').that_comes_before('Class[graphite_powershell::install]') }
+        it { should contain_class('graphite_powershell::install').that_comes_before('Class[graphite_powershell::service]') }
```

Comment:
not sure what happend here. The old syntax isn't valid. I'm not sure when this changed in the past.